### PR TITLE
Windows CI

### PR DESCRIPTION
**Pull request by filmor**

### CODE REVIEW

**Comment by filmor** [comment]
Not actually ready for review, just want to see whether the checks run this way...

**Comment by robertoaloi** [comment]
> Not actually ready for review, just want to see whether the checks run this way...

Converted it to a draft.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,51 +1,69 @@
...
+
     runs-on: ${{ matrix.platform }}
+
     container:
       image: erlang:${{ matrix.otp-version }}
+
     steps:
+    - name: Install Erlang
+      runs-on: windows-latest
+      run: choco install erlang
```
Comment (robertoaloi) [inline .github/workflows/build.yml:26]: Is the idea (in the final version) to re-use the versions of Erlang from the `otp-version` field for Windows as well?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,51 +1,69 @@
...
       uses: actions/cache@v1
       with:
         path: ~/.cache/rebar3/rebar3_*.plt
         key: ${{ runner.os }}-dialyzer-${{ hashFiles(format('{0}{1}', github.workspace, '/rebar.config')) }}
         restore-keys: |
           ${{ runner.os }}-dialyzer-
+
     - name: Compile
-      run: rebar3 compile
+      run: escript rebar3 compile
```
Comment (robertoaloi) [inline .github/workflows/build.yml:48]: Is there any way we can avoid prepending `escript`? Maybe automate the steps from https://www.rebar3.org/docs/getting-started and add them to the provisioning?

File: .github/workflows/build.yml (hunk 1 above)
Comment (filmor) [reply .github/workflows/build.yml:26]: It is, I just need to get anything to run right now...

Comment (robertoaloi) [reply .github/workflows/build.yml:26]: Sure. And thanks for the help!

**Comment by robertoaloi** [comment]
@alanz @filmor They mentioned at the Code Beam V that they started running CI for Windows for relx and see what I found:

https://github.com/erlware/relx/blob/master/.github/workflows/wintest.yml#L1

**Comment by filmor** [comment]
I'll give this another go. It doesn't use the tool-cache, yet, will look into that as well :)

**Comment by robertoaloi** [comment]
@filmor I am closing this one for inactivity (it also has conflicts). I am trying to add a simple Windows CI pipeline as well, but that seems to be more complicated than expected. PR will be opened shortly.

**Pull request closed without merging**